### DISCUSSION TITLE
Volume (and other) control for "Focused Send"

### DISCUSSION THREAD

**Discussion by CLVMTT** [helgoboss/helgobox#795] [Ideas]
In TCP, when clicking on the single send, Reaper opens up a window with all the controls for that specific Send. It would be nice to have the option to control the volume of that Focused send window, since at the moment it only allows to choose position, name and particular...

This way it would be possibile to control every send with only one fader

![Screenshot 2023-01-09 at 09 50 33](https://user-images.githubusercontent.com/92969443/211270141-364c8a56-3f4f-476c-aff0-f04f22320e16.png)

Thanks

**Comment by tompad2** [comment]
If I understand you right its already possible.
I have my X-Touch setup like then selecting button Send all available sends
on selected track is controlled by separate faders.

You find info on how to do it in Realearn forum https://forum.cockos.com/showthread.php?t=178015&page=77
RattlesnakeLodge made an example how to do it.
